### PR TITLE
Add `noarch` Version of 2.0.dev14

### PR DESCRIPTION
**Pull request by michaelweinold**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This would supersede:

 - https://github.com/conda-forge/bw2calc-feedstock/pull/15#issuecomment-1691057481

Platform-specific versions of 2.0.dev13 will be removed with:

 - https://github.com/conda-forge/admin-requests/pull/802

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: fb3e8076d6048d3acda147c83534dced9568079cf9d8d8b7fb0153731a0f8d87
+  sha256: 3b9987f0e587024787509f7826cdc9f5dcbb0fbd087d2d91503051d3aa0b9da7
 
 build:
+  noarch: python
```
Comment (cmutel) [inline recipe/meta.yaml:13]: Duplicates line below :)

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
   script: {{ PYTHON }} -m pip install . -vv
   number: 1
```
Comment (cmutel) [inline recipe/meta.yaml]: Should be reset to zero, new release

File: recipe/meta.yaml [hunk 3]
```diff
@@ -21,11 +22,11 @@ requirements:
     - setuptools
   run:
     - python >=3.4
```
Comment (cmutel) [inline recipe/meta.yaml]: Please increase this to 3.9, just because it is in the `setup.py` doesn't mean it is correct 😛

**Comment by michaelweinold** [comment]
...I need to double-check if `noarch: python` is sufficient for a `noarch` build in this case.

For the 2.0.dev13 build, [we had the same option](https://github.com/conda-forge/bw2calc-feedstock/blob/36ce0fc61df034a3beca099492e23a0e246443f2/recipe/meta.yaml#L15C3-L15C3), but still got [platform-specific builds in the channel](https://anaconda.org/conda-forge/bw2calc/files).

Am I missing something?

**Comment by michaelweinold** [comment]
@conda-forge-admin, please rerender

**Pull request merged**